homeassistant/components/home_connect/entity.py

```diff
@@ -40,9 +40,15 @@ def __init__(
         coordinator: HomeConnectCoordinator,
         appliance: HomeConnectApplianceData,
         desc: EntityDescription,
+        context_override: Any | None = None,
     ) -> None:
         """Initialize the entity."""
-        super().__init__(coordinator, (appliance.info.ha_id, EventKey(desc.key)))
+        super().__init__(
+            coordinator,
+            (appliance.info.ha_id, EventKey(desc.key))
+            if context_override is None
```

Comment:
Please avoid using ternary operator expressions that span more than one line, especially in parameter syntax where the lack of indentation then makes it heard to distinguish the parameters. I'd use a local variable that we compute first and then call super.